main.py

```diff
@@ -1,7 +1,54 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
```

Comment:
Fix these imports according to PEP8

https://peps.python.org/pep-0008/#imports